### PR TITLE
Add Transmission

### PR DESCRIPTION
**Pull request by amjed-ali-k**
qBittorent in One-click-Apps is not working for me. So I created Transmission 
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/transmission.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            PGID: '1000'
+            TZ: $$cap_tz
+            TRANSMISSION_WEB_HOME: /$$cap_ui/
+            USER: $$cap_user
+            PASS: $$cap_pass
+            WHITELIST: '' #optional
+            HOST_WHITELIST: '' #optional
+            WEBUI_PORT: '9091'
+        ports:
+            - 9091:9091
```
Comment (githubsaturn) [inline public/v4/apps/transmission.yml:16]: You probably don't need to port map 9091 as it's already being using proxied through nginx.

File: public/v4/apps/transmission.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          description: Path to pictures folder where u want to save the downloaded torrents. You can mention an existing directory here too. Eg. /home/user/torrents/
+          validRegex: '/.{1,}/'
+        - id: $$cap_ui
+          label: User Interface
+          defaultValue: transmission-web-control
+          description: Specify an alternative UI options are combustion-release, transmission-web-control, kettu and flood-for-transmission.
+          validRegex: /^(combustion-release|transmission-web-control|kettu|flood-for-transmission)$/
+    instructions:
+        start: >-
+            Transmission is designed for easy, powerful use. Transmission has the features you want from a BitTorrent client: encryption, a web interface, peer exchange, magnet links, DHT, µTP, UPnP and NAT-PMP port forwarding, webseed support, watch directories, tracker editing, global and per-torrent speed limits, and more.
```
Comment (githubsaturn) [inline public/v4/apps/transmission.yml:55]: Probably need to mention what ports are being mapped to make sure the user knows about them.
- They need to be not used.
- They need to be open and accessible from their firewall

File: public/v4/apps/transmission.yml (hunk 1 above)
Comment (amjed-ali-k) [reply public/v4/apps/transmission.yml]: sure

File: public/v4/apps/transmission.yml (hunk 2 above)
Comment (amjed-ali-k) [reply public/v4/apps/transmission.yml:54]: sure

**Comment by githubsaturn** [comment]
Thank you!

**Pull request merged**